torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8959,6 +8977,30 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    decorators=(toleranceOverride({torch.float32: tol(atol=0, rtol=4e-6), }),),
                    dtypes=all_types_and(torch.bool),
                    safe_casts_outputs=True),
+    OpInfo(
+        "nn.functional.dropout",
+        op=lambda input, *args, **kwargs:
+            wrapper_set_seed(torch.nn.functional.dropout, input, *args, **kwargs),
```

Comment:
cc @mruberry is this okay? The [documentation for `op`](https://github.com/pytorch/pytorch/blob/87d8ab6e50314f72ea6730250904fe7ff073cc21/torch/testing/_internal/common_methods_invocations.py#L469) says "the function variant of the operation, populated as torch.<name> if None". The wrapper is definitely not "the function variant of the operation".

If `op` should not be a wrapper, then we could consider something like adding a new flag to OpInfo. Something like `use_seed=None`.

Reply:
If we get to n=3 on this particular issue then I agree it's interesting to think about adding architecture to support it, but a wrapper like this seems convenient for now